recipes/example/django-controlcenter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "django-controlcenter" %}
+{% set version = "0.2.5" %}
+{% set sha256 = "3e193cce8367fc5a78ebdbae0dcd69d88ea91580a93a827ae8c28d1cb50a6a0f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/byashimov/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Would it be possible to use pip? https://conda-forge.org/docs/meta.html#use-pip

Reply:
sure. Thanks for reviewing it.